AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/AWSDataStorePlugin.swift

```diff
@@ -55,9 +55,10 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
         }
     }
 
+    #if os(watchOS)
     /// No-argument init that uses defaults for all providers
```

Comment:
This comment is not accurate for this `init`

Reply:
updating